ts_develop/conda/meta.yaml

```diff
@@ -10,7 +10,7 @@ requirements:
     - astroplan =0.8
     - astroquery =0.4
     - asynctest =0.13
-    - black =22
+    - black =23.1.0
```

Comment:
wouldn't it be better to pin it to `black=23` instead? I think we stablished in the past that pinning the major version was enough. No?

Reply:
Yeah let's give that a try. If it turns out too generous, we can always pin it to a more restrictive version.